docs/en/OLED_Display.md

```diff
@@ -0,0 +1,212 @@
+# Oled Display
+Easy extension to use for your build with OLED Display
+
+If you need help soldering, there's a small note on the end to help.
```

Comment:
I still don't like the use of word "easy". Since we're hosting this doc in our own repo, this will be us tooting our own horn essentially.

Also, since we have 2 OLED extensions, we need to explain why one would be chosen over the other, unless @xs5871 still wants to depricate the PEG OLED extension.

Reply:
I agree with that. We should at least state that this is the official and more likely to be maintained OLED extension.

Also: because this has come up before, we should at least put a little note at the top, that some kind of performance degradation has been observed / may be expected and it uses a considerable amount of RAM.